apps/ui/src/helpers/utils.ts

```diff
@@ -688,6 +696,13 @@ export function isUserAbortError(e: any) {
   );
 }
 
+export function getUserFacingErrorMessage(e: unknown): string {
+  return (
+    (e instanceof Error && e.message) ||
+    'Something went wrong. Please try again later.'
+  );
+}
+
```

Comment:
Could we pass another param, to pass a custom fallback message like "Failed to upload" like here https://github.com/snapshot-labs/sx-monorepo/pull/1426/files#r2154691321